crates/sui-core/src/consensus_handler.rs

```diff
@@ -195,8 +194,9 @@ impl<T: ParentSync + Send + Sync> ExecutionState for ConsensusHandler<T> {
                 .expect("Unrecoverable error in consensus handler");
         }
 
+        let round = consensus_output.sub_dag.round();
```

Comment:
Maybe add a quick comment about this being the leader round?

Reply:
I have better idea - renaming SubDag::round to SubDag::leader_round